README.md

```diff
@@ -33,4 +37,29 @@ Please see the [contribution guidelines](CONTRIBUTING.md).
     - Exit Survey
   - Learning & Development
     - [Training](docs/training/training-toc.md)
+- Products
+  - DreamBig
+  - OnTrack
+  - Splashkit
+- Projects
+  - T1 2022
+    - [Teams](docs/projects/t1-2022/teams.md)
+    - Documentation
+    - DreamBig Prototype
+    - Internal Systems
+    - OnTrack Deployment
+    - OnTrack Front-end Migration
+    - OnTrack Jupyter Notebook Support
+    - OnTrack Voice Verification
+    - Splashkit Applications
+    - Splashkit Extensions
+    - Splashkit Modules
+    - Splashkit Operations
+    - [Thoth Tech Leadership](docs/projects/t1-2022/thoth-tech-leadership)
+      - Retrospectives
+      - [Weekly Announcements](docs/projects/t1-2022/thoth-tech-leadership/weekly-announcements/)
+  - T2 2022
 - Tools
+- Training
+  - [Markdown Guide](docs/training/markdown-guide.md)
+  - [Training Guide](docs/training/training-guide.md)
```

Comment:
**suggestion:** Since we already have the _index_/landing page for training, I don't think we need to repeat the structure. What do you think?

```suggestion
- [Training](docs/training/training-guide.md)
```